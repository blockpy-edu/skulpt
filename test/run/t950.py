# Test basic match statement functionality
x = 5
match x:
    case 1:
        print("one")
    case 5:
        print("five")
    case _:
        print("default")

# Test string matching
s = "hello"
match s:
    case "hello":
        print("greeting")
    case "world":
        print("planet")
    case _:
        print("unknown")

print("match tests completed")