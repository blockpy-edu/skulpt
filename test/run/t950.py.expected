five
greeting
match tests completed